opentelemetry-instrumentation/README.rst

```diff
@@ -71,13 +72,14 @@ Well known trace exporter names:
     - jaeger_proto
     - jaeger_thrift
     - opencensus
-    - otlp
-    - otlp_proto_grpc
-    - otlp_proto_http
     - zipkin_json
     - zipkin_proto
+    - otlp
+    - otlp_proto_grpc (`deprecated`)
+    - otlp_proto_http (`deprecated`)
```

Comment:
I think we should not recommend using this, as the defined way in OTel to configure the protocol is with the other env vars, and to avoid conflicts.

In future versions, we may also remove these options (though it's a breaking change).